src/components/views/PlayWhites.js

```diff
@@ -4,52 +4,144 @@ import BaseContainer from "components/ui/BaseContainer";
 import "styles/views/WhiteCardSelection.scss";
 import Header from "./Header";
 import CardButton from "../ui/CardButton";
-import MenuContainer from "../ui/MenuContainer";
+import {api, handleError} from "../../helpers/api";
+import PropTypes from "prop-types";
+
+//TODO: fetch a game of a random user (todo) and your white cards (done)
 
 
 const PlayWhites = () => {
   // use react-router-dom's hook to access the history
   const history = useHistory();
+  const[blackCard, setBlackCard] = useState(null)
+  const[gameId, setGameId] = useState(null)
+  const [cards, setCards] = useState(null)
+    const [count, setCount] = useState(0)
+  const userId = localStorage.getItem("id")
+  const token = localStorage.getItem("token")
+
+    const WhiteCard = ({card}) => {
+        // use react-router-dom's hook to access the history
+        const history = useHistory();
+
+        // put the black Card to the Server and proceed to main menu
+        const selectCard = async ()  => {
+            // TODO: onClick: ask to confirm this card
```

Comment:
use window.confirm(), it is also used in Edit User Page Line 150.

Reply:
done